docs/releases/v8_14.md

```diff
@@ -0,0 +1,48 @@
+# Release Notes
+
+## February 2023 (version 8.14)
+
+### Overview
+
+The **February 11th, 2023** release of **DietPi v8.14** comes with a new image for the Radxa **ROCK 5B**, **Odroid N2/HC4**, **Orange Pi 5** and a couple of improvements and bug fixes.
+
+![Icy bike](../assets/images/dietpi-release-v8_14.jpg){: width="480" height="331" loading="lazy"}
+
+!!! cite "*Photo by `borismayer77`, Pixabay*"
+
+### New images
+
+- [**Orange Pi 5**](../../hardware/#orange-pi-series) :octicons-arrow-right-16: Support for Orange Pi 5 SBC has been added to DietPi with hardware ID 80.
+- [**Odroid N2/C4/HC4**](../../hardware/#odroid) :octicons-arrow-right-16: New images have been generated with support for the default Hardkernel SPI bootloader petitboot. Thanks to this, boot from USB devices is now possible, and on Odroid HC4 it is not necessary anymore to erase the SPI flash or flash mainline U-Boot to it. Many thanks to @hstarcm and @Power-onoff for bringing up this topic and help testing adjustments to the boot script: <https://github.com/MichaIng/DietPi/issues/5634>
+- [**ROCK 5B**](../../hardware/#radxa) :octicons-arrow-right-16: New images with ext4 boot filesystem have been generated, to solve possible issues and limitations of the previously used FAT filesystem. Many thanks to @isarrider for reporting this issue: <https://github.com/MichaIng/DietPi/issues/6122>
+
+### Improvements
+
+- **WiFi** :octicons-arrow-right-16: Since Linux 4.15, the kernel can read the WiFi country code from connected access points, and searches the regulatory database for related allowed frequencies automatically. The Central Regulatory Domain Agent (CIDR) has hence become obsolete and isn't even functional if the kernel feature is used. Linux needs to have the regulatory database in specific format stored below /lib/firmware for this, which is the case since Debian Bullseye. Additionally, if no Debian kernel is used (all SBCs), one needs to change the used regulatory database from a Debian-only one to the upstream one. New DietPi images are shipped now with this setup without CRDA, it is applied on next DietPi update on Bullseye and Bookworm systems, and it is offered to purge the obsolete CRDA package. This also fixes the issue on Bookworm SBC systems, where the limited global (00) country code was used, since CRDA is not available on Bookworm, but the Debian-only regulatory database used by default.
```

Comment:
```suggestion
- **WiFi** :octicons-arrow-right-16: Since Linux 4.15, the kernel can read the WiFi country code from connected access points, and searches the regulatory database for related allowed frequencies automatically. The Central Regulatory Domain Agent (CRDA) has hence become obsolete and isn't even functional if the kernel feature is used. Linux needs to have the regulatory database in specific format stored below /lib/firmware for this, which is the case since Debian Bullseye. Additionally, if no Debian kernel is used (all SBCs), one needs to change the used regulatory database from a Debian-only one to the upstream one. New DietPi images are shipped now with this setup without CRDA, it is applied on next DietPi update on Bullseye and Bookworm systems, and it is offered to purge the obsolete CRDA package. This also fixes the issue on Bookworm SBC systems, where the limited global (00) country code was used, since CRDA is not available on Bookworm, but the Debian-only regulatory database used by default.
```